modules/plugins/wfly-10/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -8709,245 +8709,251 @@
-                <c:option value="TLSv1"/>
-                <c:option value="SSLv2+SSLv3"/>
-                <c:option value="ALL"/>
-              </c:property-options>
-            </c:simple-property>
-            <c:simple-property name="session-cache-size" required="false" type="integer" readOnly="false" description="SSL session cache."/>
-            <c:simple-property name="session-timeout" required="false" type="integer" readOnly="false" description="SSL session cache timeout."/>
-            <c:simple-property name="truststore-type" required="false" type="string" readOnly="false" description="Type of the truststore, There are various types of keystores available, including &apos;PKCS12&apos; and Sun&apos;s &apos;JKS.&apos;"/>
-            <c:simple-property name="verify-client" required="false" type="string" readOnly="false" description="Enable client certificate verification."/>
-            <c:simple-property name="verify-depth" required="false" type="integer" readOnly="false" description="Limit certificate nesting."/>
-          </c:map-property>
-        </c:group>
+          <c:simple-property name="default-security-domain" required="false" type="string" readOnly="false" defaultValue="other"
+         description="The default security domain used by web deployments. The default value is other."/>
+          <c:simple-property name="default-server" required="false" type="string" readOnly="false" defaultValue="default&#45;server"
+         description="The default server to use for deployments. The default value is default&#45;server."/>
+          <c:simple-property name="default-servlet-container" required="false" type="string" readOnly="false" defaultValue="default"
+         description="The default servlet container to use for deployments. The default value is default."/>
+          <c:simple-property name="default-virtual-host" required="false" type="string" readOnly="false" defaultValue="default&#45;host"
+         description="The default virtual host to use for deployments. The default value is default&#45;host."/>
+          <c:simple-property name="instance-id" required="false" type="string" readOnly="false" defaultValue="{EXPRESSION_VALUE=&#36;{jboss.node.name}}"
+         description="The cluster instance id. The default value is {EXPRESSION_VALUE=&#36;{jboss.node.name}}."/>
+          <c:simple-property name="statistics-enabled" required="false" type="boolean" readOnly="false" defaultValue="false"
+         description="Configures if are statistics enabled. The default value is false."/>
       </resource-configuration>
-
-    </service>
-
-    <service name="VHost"
-             discovery="SubsystemDiscovery"
-             class="VHostComponent"
-             createDeletePolicy="both">
+      
+      <service name="Buffer Cache"
+                            discovery="SubsystemDiscovery"
+                            class="BaseComponent"
+                            createDeletePolicy="both">
```

Comment:
The CLI had the "add" and "delete" operation on that node, that's why i though it could be possible. Is that enough to decide if we should allow adding / deleting ? If not I'll remove the createDeletePolicy once we get more information.